TA_dataset/bin/dataset_api.py

```diff
@@ -242,7 +245,7 @@ def parse_query(ds_columns, match_list, sessions):
     if ds_columns is None:
         session_key = match_list["session"]
 
-        for session_entry, session_dict in sessions.items():
+        for session_entry, session_dict in list(sessions.items()):
```

Comment:
Does the `list` conversion necessary here?

Reply:
this is something the linter is made as a suggestion